src/screens/Services/Services.js

```diff
@@ -253,22 +247,7 @@ class ServicesScreen extends React.Component<Props> {
           const email = user?.email;
           const address = this.getCryptoPurchaseAddress();
           if (address === null) return;
-
-          Alert.alert(
-            t('servicesContent.ramp.assetDecisionAlert.title'),
-            t('servicesContent.ramp.assetDecisionAlert.description'),
-            [
-              {
-                text: t('servicesContent.ramp.assetDecisionAlert.actionNonPlr'),
-                onPress: () => this.tryOpenCryptoPurchaseUrl(rampWidgetUrl(address, email)),
-              },
-              {
-                text: t('servicesContent.ramp.assetDecisionAlert.actionPlr'),
-                onPress: () => this.tryOpenCryptoPurchaseUrl(rampWidgetUrl(address, email, true)),
-              },
-            ],
-            { cancelable: true },
-          );
+          navigation.navigate(ADD_CASH, { address, email });
```

Comment:
Pls make ADD_CASH work without these params: add cash screen should be able to query them from redux by itself. This is to allow easy navigation from other parts of the app.

Reply:
Done.